docs/3.0.0-beta.x/concepts/controllers.md

```diff
@@ -17,11 +17,13 @@ In this example, any time a web browser is pointed to the `/hello` URL on your a
 
 ### Where are the controllers defined?
 
-The controllers are defined in each `./api/**/controllers/` folder. Every JavaScript file put in these folders will be loaded as a controller. They are also available through the `strapi.controllers` and `strapi.api.**.controllers` global variables. By convention, controllers' names should be Pascal-cased, so that every word in the file (including the first one) is capitalized `User.js`, `LegalEntity.js`.
+The controllers are defined in each `./api/**/controllers/` folder. Every JavaScript file put in these folders will be loaded as a controller. They are also available through the `strapi.controllers` and `strapi.api.**.controllers` global variables.
+
+By convention, controllers' names should be Pascal-cased, so that every word in the file (including the first one) is capitalized `User.js`, `LegalEntity.js`.
```

Comment:
That convention should be removed.

Reply:
Hum so what should I do exactly?

Reply:
What is the new format?

Reply:
it didn't change the in code it is going to be lowercased so the name of the file doesn't matter :) you can remove the line